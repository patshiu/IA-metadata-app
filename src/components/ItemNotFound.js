import React from 'react';
import PropTypes from 'prop-types';

const ItemNotFound = ({ itemIdentifier }) => {
  return (
    <div className="row center">
      <h1>Unable to find the item &apos;{itemIdentifier}&apos;.</h1>
    </div>
  );
};

ItemNotFound.propTypes = {
  itemIdentifier: PropTypes.string.isRequired
};

export default ItemNotFound;
