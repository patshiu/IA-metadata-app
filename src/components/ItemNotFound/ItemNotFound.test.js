import React from 'react';
import renderer from 'react-test-renderer';

import ItemNotFound from './ItemNotFound';

it('renders item not found notice correctly', () => {
  const tree = renderer.create(<ItemNotFound itemIdentifier="PhantomItem" />).toJSON();
  expect(tree).toMatchSnapshot();
});
