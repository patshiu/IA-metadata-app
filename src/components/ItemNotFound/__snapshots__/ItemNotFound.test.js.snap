// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders item not found notice correctly 1`] = `
<div
  className="row center"
>
  <h1>
    Unable to find the item '
    PhantomItem
    '.
  </h1>
</div>
`;
